Tools/AP_Bootloader/board_types.txt

```diff
@@ -79,6 +79,7 @@ AP_HW_MRO_CONTROL_ZERO_OEM            142
 AP_HW_MATEKF765_WING                  143
 AP_HW_JDMINIF405                      144
 AP_HW_KAKUTEF7_MINI                   145
+AP_HW_KAKUTEF4_MINI                   146
```

Comment:
why this ID? why not one in the ardupilot range?

Reply:
Sorry did not know that range was reserved, fixed now